ironfish/src/primitives/target.ts

```diff
@@ -110,6 +115,8 @@ export class Target {
     targetBlockTimeInSeconds: number,
     targetBucketTimeInSeconds: number,
     maxBuckets: number,
+    enableFishHashSequence: ActivationSequence,
```

Comment:
could combine these into one param if we want to check sequence outside of this function